docs/getting-started.md

```diff
@@ -161,83 +194,147 @@ First we need to setup CLion
+            - Pick the network interface you would like to use:
+                1. If you are running things locally, you can pick any interface that is not `lo`
+                2. If you would like to communicate with robots on the network, make sure to select the interface that is connected to the same network as the robots.
+            - For example, on a sample machine, the output may look like this:
+
+                ```
+                enp0s5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+                        ...
+                        [omitted]
+                        ...
+
+                lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
+                        ...
+                        [omitted]
+                        ...
+                ```
+
+            - An appropriate interface I could choose is `enp0s5`
+        - If we are running the AI as "blue": `./tbots.py run thunderscope_main --interface=[interface_here] --run_blue`
+        - If we are running the AI as "yellow": `./tbots.py run thunderscope_main --interface=[interface_here] --run_yellow`
+        - `[interface_here]` corresponds to the `ifconfig` interfaces seen in the previous step
+            - For instance, a call to run the AI as blue on wifi could be: `./tbots.py run thunderscope_main --interface=enp0s5 --run_blue`
+        - This command will set up robot communication and the Unix full system binary context manager. The Unix full system context manager hooks up our AI, Backend and SensorFusion
+2. Run Robot Diagnostics:
+    - (#2711) There isn't a clean way to do this at the moment.
     - The Mechanical and Electrical sub-teams use Robot Diagnostics to test specific parts of the Robot.
-6. Run our SimulatedPlayTests in the visualizer: `bazel test //software/ai/hl/stp/play:[some_target_here] --test_arg="--enable_visualizer"` or `bazel run //software/ai/hl/stp/play:[some_target_here] -- --enable_visualizer`
+3. Run our SimulatedPlayTests in Thunderscope
     - This will launch the visualizer and simulate AI Plays, allowing us to visually see the robots acting according to their roles.
-7. Run our SimulatedTacticTests in the visualizer: `bazel test //software/ai/hl/stp/tactic:[some_target_here] --test_arg="--enable_visualizer"` or `bazel run //software/ai/hl/stp/tactic:[some_target_here] -- --enable_visualizer`
-    - This will launch the visualizer and simulate AI Tactic on a single robot
-
-** NOTE: If you want to run SimulatedTests with the AI initially stopped, then use the `--stop_ai_on_start` flag ** 
-
-### Running AI vs AI
-1. Open your terminal, `cd` into `Software/src`
-2. Run `./tbots.py run thunderscope`
+    1. For legacy C++ tests (#2581) with the visualizer:
+        1. First run Thunderscope configured for receiving protobufs over unix sockets correctly: `./tbots.py run thunderscope_main -- --visualize_cpp_test`
+        2. Then run `./tbots.py test [some_target_here] -- --run_sim_in_realtime`
```

Comment:
nit: I dont think you need the first `--`

Reply:
updated